e2etests/file_test.go

```diff
@@ -0,0 +1,140 @@
+package e2etests
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"io"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"github.com/brancz/locutus/render"
+	"github.com/efficientgo/e2e"
+	"github.com/efficientgo/tools/core/pkg/errcapture"
+	"github.com/efficientgo/tools/core/pkg/testutil"
+	"github.com/pkg/errors"
+)
+
+const locutusImage = "quay.io/brancz/locutus:latest"
+
+// TestFileExampleRender tests locutus in docker rendering of examples in `/example/files` directory.
+// This e2e tests require `make docker` first.
+func TestFileExampleRender(t *testing.T) {
+	e, err := e2e.NewDockerEnvironment("e2e_file")
```

Comment:
Does this really need to run in docker?

Reply:
I get that an e2e test for the kubernetes functionality should run on Kubernetes, but everything else should just be plainly run on the machine, no?

Reply:
Yea, well, my arguments were:

* The direct usage of Locutus is primarily through containers, so we test more by using the docker image directly
* We have a ready testing framework for docker that is consistently used across all our project
* Running full Kubernetes just to test Locutus sounds like overkill, plus it's hard to find CI for running Kubernetes for tests purposes (reliable one and ideally free/cheap). 

But yea, happy to discuss - you would prefer using `os.Exec`?

Reply:
yeah I think I would prefer os.Exec